TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/HistoryController.php

```diff
@@ -40,9 +44,10 @@ class HistoryController extends AbstractModuleController
      */
     public function indexAction($offset = 0, $limit = 10)
     {
-        $events = $this->eventRepository->findRelevantEvents($offset, $limit + 1)->toArray();
+        $events = $this->eventRepository->findRelevantEventsByWorkspace($offset, $limit + 1, 'live')->toArray();
```

Comment:
Good one, when was that added? That filtering further down existed for ages already…